src/native/util/common.h

```diff
@@ -29,18 +30,37 @@ namespace noobaa
 #define DVAL(x) #x "=" << x << " "
 
 #define LOG(x) std::cerr << LOG_PREFIX() << x << std::endl
+#define napi_log(x)                                    \
+    do {                                               \
+        int facility = LOG_LOCAL0;                     \
+        int priority = 0;                              \
+        ::syslog(priority | facility, "%s", x);        \
+    } while (0)
 
+#define SYS_LOG(x)                                          \
+    do {                                                    \
+            std::ostringstream oss;                         \
+            oss << "" << LOG_PREFIX() << x << std::endl;    \
+            std::string var1 = oss.str();                   \
+            const char* cstr1 = var1.c_str();               \
+            napi_log(cstr1);                                \
+    } while (0)
 // to use DBG the module/file should use either DBG_INIT or DBG_INIT_VAR.
 #define DBG_INIT(level) static int __module_debug_var__ = level
 #define DBG_INIT_VAR(debug_var) static int& __module_debug_var__ = debug_var
 #define DBG_SET_LEVEL(level) __module_debug_var__ = level
 #define DBG_GET_LEVEL() (__module_debug_var__)
 #define DBG_VISIBLE(level) (level <= __module_debug_var__)
-#define DBG(level, x)                        \
-    do {                                     \
-        if (DBG_VISIBLE(level)) {            \
-            LOG("[L" << level << "] " << x); \
-        }                                    \
+#define DBG(level, x)                                                   \
+    do {                                                                \
+        if (DBG_VISIBLE(level)) {                                       \
+            const char* syslog_enabled = std::getenv("SYSLOG_ENABLED"); \
```

Comment:
could you move it to be a global?

Reply:
You mean `SYSLOG_ENABLED`?

Reply:
yes, like we have
const char* gpfs_dl_path = std::getenv("GPFS_DL_PATH");